Tests/BackendTest/StudentServiceTest.cs

```diff
@@ -20,6 +21,32 @@ public StudentServiceTest(Mongo2GoFixture mongoFixture)
             _mongoFixture = mongoFixture;
         }
 
+        
+        [Fact]
+        public async Task AssingTimetableToStudent()
+        {
+            IMongoDatabase database = _mongoFixture.MongoClient.GetDatabase("StudentsDB");
+            StudyGroupService grpsrvc = new StudyGroupService(database);
+            CourseService crsrv = new CourseService(database);
+            ISchoolScheduleProxy proxy = new FakeProxy();
+
+            StudyGroup sg = await grpsrvc.GetStudyGroupAsync("5ZI001", crsrv, proxy);
+            Student s = new Student();
```

Comment:
I know it is just test, but we have to uses meaningful names for variables, not just 's',..